connectivity/mbedtls/tools/importer/adjust-config.sh

```diff
@@ -117,6 +117,9 @@ conf unset MBEDTLS_SSL_TRUNCATED_HMAC
 
 conf unset MBEDTLS_PLATFORM_TIME_TYPE_MACRO
 
+# not needed for production code
```

Comment:
Some production code might use self-tests to ensure the hardware is working correctly before continuing with other functional stages of the application. For example, your PC's BIOS likely does some self-tests before continuing with the boot. We should put the reason as something more like "potentially save flash space by not enabling self-tests by default".

Reply:
Done, sounds good. It looks like the linker does filter out most unused functions, but some static variables are still linked to the executable. Also seem this happening to BLE.